libs/platform/user/ebpf_handle_user.c

```diff
@@ -70,7 +70,6 @@ ebpf_handle_create(ebpf_handle_t* handle, ebpf_object_t* object)
 ebpf_result_t
 ebpf_handle_close(ebpf_handle_t handle)
 {
-    EBPF_LOG_ENTRY();
```

Comment:
were these removed for perf reasons?  In other places you put a comment about to tracing due to perf.